exoscale/resource_exoscale_ipaddress_test.go

```diff
@@ -48,6 +48,7 @@ func TestAccResourceIPAddress(t *testing.T) {
 						"healthcheck_timeout":      ValidateString(fmt.Sprint(testIPHealthcheckTimeout1)),
 						"healthcheck_strikes_ok":   ValidateString(fmt.Sprint(testIPHealthcheckStrikesOk1)),
 						"healthcheck_strikes_fail": ValidateString(fmt.Sprint(testIPHealthcheckStrikesFail1)),
+						"description":              ValidateString("IPAdress 1"),
```

Comment:
Please don't hardcode test values but rather follow the existing pattern of declaring constants (e.g. `testIPDescription1 = "IP address 1"`) and reference them throughout the tests: https://github.com/pierre-emmanuelJ/terraform-provider-exoscale/blob/ch6590/eip-add-description/exoscale/resource_exoscale_ipaddress_test.go#L13-L28

Reply:
thx